scripts/variantstore/wdl/GvsCreateVDS.wdl

```diff
@@ -215,7 +217,7 @@ task CreateVds {
             minInstances: 2
             maxInstances: 2
         secondaryWorkerConfig:
-            maxInstances: 500
+            maxInstances: 200
```

Comment:
this seemed excessively high and may have been contributing to some of our stability issues at scale; happy to discuss